src/components/microlearning/styles/VideoDetailPage.scss

```diff
@@ -0,0 +1,99 @@
+.video-container {
+  gap: 16px;
+  flex: 1 0 0;
+}
+
+.video-skills {
+  padding: 8px;
+  margin-right: 7px;
+  margin-top: 8px;
+  font-size: 12.5px;
+  font-weight: normal;
+  gap: 16px;
+}
+
+.video-wrapper {
+  position: relative;
+  max-width: 100%;
+  overflow: hidden;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
+
+.video-wrapper .video-js-wrapper {
+  position: relative;
+  padding-top: 56.25%;
+}
+
+.video-wrapper .video-js-wrapper .video-js {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100% !important;
+}
+
+.vjs-control-bar .vjs-playback-rate .vjs-playback-rate-value {
+  font-size: 13px;
+  margin-top: 2px;
+}
+
+.video-player-container-with-transcript {
+  display: flex;
+}
+
+.video-js-wrapper {
+  display: flex;
+  flex-direction: row; 
+  align-items: flex-start; 
+  width: 100%; 
+  max-width: 100%; 
+}
+
+.video-js-wrapper .video-js {
+  max-width: 100%; 
+}
+
+
+.transcript-container {
+  max-width: 30% !important;
+  height: 100%;
+  max-height: 100%;
+}
+
+.vjs-transcribe-body {
+  padding: 40px !important;
+  background: #f9f9f9; 
+  border: 1px solid #ddd; 
+  max-height: 545px !important;
+  height: 100% !important;
+  font-size: 14px;
+  line-height: 24px;
+}
+
+.vjs-transcribe-pip {
+  position: static !important;
+  bottom: 1rem;
+  left: 1rem;
+  display: none;
+  max-width: 500px;
+  width: calc(100% - 2rem);
+  background: red;
+  z-index: 100;
+}
+
+.vjs-transcribe-search {
+  display: none;
+}
+
+.vjs-transcribe-btn .vjs-transcribe-copy {
+  display: none;
+}
+
+.vjs-transcribe-btn {
+  display: none;
+}
+
+.vjs-transcribe-cueline {
+    color: #00688D;
+}
```

Comment:
Most of this CSS already has classes available. We should use them as much as possible. I'm suggesting this because Adam Stankiewicz asked our team to follow this practice.

Reason: Custom CSS could cause theming issues in future.

Reply:
@jajjibhai008 We are targeting the plugin's hidden elements here, so it is not possible to use CSS classes.